allure-generator/src/main/javascript/components/tree/TreeView.js

```diff
@@ -50,25 +40,16 @@ class TreeView extends View {
         }
     }
 
-    onBeforeRender() {
-        const visibleStatuses = this.settings.getVisibleStatuses();
-        const filter = byStatuses(visibleStatuses);
-
-        const sortSettings = this.settings.getTreeSorting();
-        const sorter = getComparator(sortSettings);
-
-        this.collection.applyFilterAndSorting(filter, sorter);
-    }
 
     onRender() {
         this.selectNode();
-        this.showChildView('sorter', new NodeSorterView({
-            settings: this.settings
-        }));
-        this.showChildView('filter', new StatusToggleView({
-            settings: this.settings,
-            statistic: this.collection.statistic
-        }));
+        if (this.searchQuery) {
+            this.$('.node__title').each((i, node) => {
+                this.$(node).parent().addClass('node__expanded');
+            });
```

Comment:
Something weird is happening here. 
@baev do we have expanded state stored in model?

Reply:
yes, when user clicks on a node, its state is stored, but i decided that better way is to force opening all nodes when search is in progress

Reply:
when search is finished, i restore previous state.
I already see bug here – if user toggles nodes by click while he is in search mode, this state is saved and then will be restored incorrectly.